drm.c doesn't build, not needed for qemu-img so just avoid building it

--- qemu-4.1.0/util/Makefile.objs	2019-08-15 19:01:43.000000000 +0000
+++ new/qemu-4.1.0/util/Makefile.objs	2019-11-14 20:43:36.541907420 +0000
@@ -52,7 +52,6 @@
 util-obj-y += iova-tree.o
 util-obj-$(CONFIG_INOTIFY1) += filemonitor-inotify.o
 util-obj-$(CONFIG_LINUX) += vfio-helpers.o
-util-obj-$(CONFIG_POSIX) += drm.o
 util-obj-y += guest-random.o
 
 stub-obj-y += filemonitor-stub.o
